DQM/GEM/plugins/GEMDQMHarvester.cc

```diff
@@ -273,7 +271,8 @@ void GEMDQMHarvester::drawSummaryHistogram(edm::Service<DQMStore> &store, Int_t
   for (const auto &strSuffix : listLayer_) {
     if (mapIdxLayer_.find(strSuffix) == mapIdxLayer_.end())
       continue;
-    auto nNumChamber = mapNumChPerChamber_[mapIdxLayer_[strSuffix]];
+    //auto nNumChamber = mapNumChPerChamber_[mapIdxLayer_[strSuffix]];
```

Comment:
Hi @tvami, 

I know it's not recommended to leave a comment line like that, but I left the line for a reminder since the following line, which replaces the commented line, is just a temporary fix. And I think the general solution would be something like the previous version.
But I'll remove it if it's still not a good idea.